src/main/java/org/datadog/jenkins/plugins/datadog/publishers/DatadogComputerPublisher.java

```diff
@@ -80,33 +79,41 @@ protected void doRun() throws Exception {
             // Add JenkinsUrl Tag
             globalTags = TagsUtil.addTagToTags(globalTags, "jenkins_url", DatadogUtilities.getJenkinsUrl());
             for (Computer computer : computers) {
+                Map<String, Set<String>> tags = TagsUtil.merge(
+                        DatadogUtilities.getComputerTags(computer), globalTags);
                 nodeCount++;
+                boolean online = true;
```

Comment:
any reason to keep this variable? what if the metric is submitted during the online/offline checks?

```
if (computer.isOffline()) {
     nodeOffline++;
      client.gauge("jenkins.node_status.up", 0, hostname, tags);
}	          
 if (computer.isOnline()) {	        
     nodeOnline++;
      client.gauge("jenkins.node_status.up", 1, hostname, tags);
}	              

```

Reply:
Sure, that way is simpler now that there's only one metric!

Reply:
Updated!